code/mob/living/carbon/human.dm

```diff
@@ -3190,6 +3190,12 @@
 		return
 	src.chest_item.AttackSelf(src)
 
+///Clear chest item if it escapes/gets disposed
+/mob/living/carbon/human/Exited(atom/movable/thing)
+	if (thing == chest_item)
```

Comment:
Tiny gripe- should call the parent here
doesn't do anything currently but might in the future
```suggestion
/mob/living/carbon/human/Exited(atom/movable/thing)
	..()
	if (thing == chest_item)
```